contracts/Aavegotchi/facets/BridgeFacet.sol

```diff
@@ -89,6 +90,7 @@ contract BridgeFacet is Modifiers {
             for (uint256 i; i < tokenIds.length; i++) {
                 uint256 tokenId = tokenIds[i];
                 require(address(this) == s.aavegotchis[tokenId].owner, "Bridge: Not owner of token");
+                LibAavegotchiLending.enforceAavegotchiNotInRental(tokenId);
```

Comment:
I don't understand why this is here.   This code is executed when an Aavegotchi is transferred from Ethereum to Polygon.  It isn't possible for an Aavegotchi to be on Ethereum and rented on Poygon at the same time.

Reply:
OK, Let me remove.

Reply:
What about preventing people from bridging their Aavegotchi from Polygon to Ethereum (locking it into the Diamond). Is there anything preventing this behavior? @orionstardust

Reply:
@cinnabarhorse Aavegotchi transfer is disabled during rental. So users are unable bridging gotchies.

Reply:
Keep in mind that it is only transfers that use `internalTransferFrom` that are disabled. So don't assume that all transfers are disabled.  The bridge doesn't use that function to transfer Aavegotchis, however the bridge does not allow locked Aavegotchis to be bridged which prevents Aavegotchis from being bridged when rented.

Reply:
Yes I see in BridgeFacet that transferring is disabled for locked gotchis, and renting locks gotchis. So no issue there.